backend/core/tests/management/commands/test_populateperiodictasks.py

```diff
@@ -0,0 +1,12 @@
+from django_celery_beat.models import PeriodicTask
+from django.core.management import call_command
+from django.test import TestCase
+
+
+class ClosepollTest(TestCase):
```

Comment:
Shouldn’t the class name be `PopulatePeriodicTasksTest`?

Otherwise all good, thanks!